### PR TITLE
Query and update animals table

### PR DESCRIPTION
**Pull request by yosaddis**
In this feature I completed the day two projects by updating the following files
-Updated query.sql 
-Update schema.sql 
-Updated data.sql
Please see the screenshot for detail.
1- Alter animal table
2- insert statement
3- update species column
4-update the species value
5- delete and rollback delete
6- Savepoint
7- count quires

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -12,3 +12,19 @@ VALUES
...
+    weight_kg
+  )
+VALUES
+  ('Charmander', '2020-02-08', 0, false, 11),
+  ('Plantmon', '2021-11-15', 2, true, 5.7),
+  ('Squirtle', '1993-04-02', 3, false, 12.13),
+  ('Angemon', '2005-06-12', 1, true, 45),
+  ('Boarmon', '2005-06-07', 7, true, 20.4),
+  ('Blossom', '1998-10-13', 3, true, 17),
+  ('Ditto', '2022-05-14', 4, true, 22);
```
Comment (KDlamini) [inline data.sql]: Kindly make sure that you include the negative weights for `Charmander`, `Plantmon`, `Squirtle`, and `Angemon` as in the project requirements. The negative weights are not a mistake, they are there on purpose.

They will later be corrected with the transaction query in the `querries.sql` file:
```sql
    UPDATE animals SET weight_kg = weight_kg * -1; 
    ROLLBACK TO SP1;
    UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0; 
```

File: queries.sql [hunk 2]
```diff
@@ -5,4 +5,110 @@ SELECT date_of_birth FROM animals WHERE name='Agumon' OR name='Pikachu';
...
+  species;
+SELECT
+  species,
+  AVG(escape_attempts)
+FROM
+  animals
+WHERE
+  date_of_birth BETWEEN '1990-01-01'
+  AND '2000-01-01'
+GROUP BY
```
Comment (KDlamini) [inline queries.sql]: Kindly make sure that the selection includes the whole year of `2000`. you are only selecting up to 1st January 2000 and therefore excluding the days and months from 2nd January to December 31, 2000.

- The date selection range should be up to `2000/12/31`

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @yosaddis 🙋‍♀️,  

Great work so far! 💯 
You have done a great job querying and updating the animal's table 🎊 🥳 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  Good PR title and description
✔️  All SQL commands schema, data, and queries are added correctly

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
Hi @yosaddis, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯
3. Well-written code 💯 

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**